src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -107,16 +135,28 @@ public void verifyUserIsVisibleOnContactsList(String userName) {
         + " is visible on contacts list", true, driver);
   }
 
-  public void verifyPrivateMessageHeader() {
-    wait.forElementVisible(privateMessagesHeader);
-    PageObjectLogging.log("verifyPrivateMessageHeader", "private message header is visible", true,
-        driver);
+  public boolean isPrivateMessageHeaderDispayed() {
+    try {
+      return privateMessagesHeader.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
   }
 
-  public void verifyPrivateMessageNotification() {
-    wait.forElementVisible(privateMessageNotification);
-    PageObjectLogging.log("verifyPrivateMessageNotification",
-        "private message notification is visible", true, driver);
+  public boolean isPermissionsErrorTitleDisplayed() {
+    try {
+      return permissionsErrorTitle.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isPrivateMessageNotificationDisplayed() {
+    try {
+      return privateMessageNotification.isDisplayed();
+    }catch (ElementNotFoundException e){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)